src/config/konnectors.json

```diff
@@ -1561,6 +1561,32 @@
       "bankId": "119"
     }
   },
+  {
+    "slug": "impots",
+    "name": "Impots.gouv",
+    "editor": "Cozy Cloud",
+    "vendorLink": "https://cfspart.impots.gouv.fr/",
+    "categories": ["public_service"],
+    "dataType": [
+      "public_service"
+    ],
+    "fields": {
+      "identifier": {
+        "type": "text"
```

Comment:
You could not know but you can specify some limits to the fields. Here I would do : 

```js
      "login": {
        "type": "text",
        "min": 13,
        "max": 13
      },

```

Reply:
Good to know, is there some documentation for this?